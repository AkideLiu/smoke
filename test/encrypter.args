fixtures/encrypter.rb
fixtures/encrypter
