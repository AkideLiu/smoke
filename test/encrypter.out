color
[31m  input:    "[0mthis is a [32mgreen[0m word[31m"
  output:   "[0mguvf vf n [32mterra[0m jbeq[31m"
  expected: "[0muijt jt b [32mhsffo[0m xpse[31m"[0m
lines
[31m  input:    "one\ntwo\nthree\nfour"
  output:   "bar\ngjb\nguerr\nsbhe"
  expected: "pof\nuxp\nuisff\ngpvs"[0m
word
[31m  input:    "word"
  output:   "jbeq"
  expected: "xpse"[0m

[31m3 tests, 3 failures[0m
